tools/SourceKit/tools/sourcekitd/include/sourcekitd/Internal.h

```diff
@@ -34,6 +34,16 @@ namespace SourceKit {
   class UIdent;
 }
 
+SOURCEKITD_PUBLIC SOURCEKITD_NONNULL_ALL bool
+sourcekitd_variant_dictionary_apply_impl(
+  sourcekitd_variant_t dict,
+  llvm::function_ref<bool(sourcekitd_uid_t, sourcekitd_variant_t)> applier);
+
+SOURCEKITD_PUBLIC SOURCEKITD_NONNULL_ALL bool
```

Comment:
`SOURCEKITD_PUBLIC` should not be used here since these are not part of the public C API, and `SOURCEKITD_NONNULL_ALL` is unnecessary because the parameters are not pointer types.